ide/editor.lib/apichanges.xml

```diff
@@ -83,6 +83,21 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="caret-min-width">
+            <summary>Allow to specify minimum caret width</summary>
+            <version major="4" minor="18"/>
```

Comment:
The module version shall increase in its manifest and this element shall refer to the increased version.